frontend/src/components/organisms/SignupForm.tsx

```diff
@@ -66,6 +66,12 @@ const SignupForm = () => {
     ) : null;
   };
 
+  const CreateSuccessComponent = (): JSX.Element | null => {
+    return errorMessage ? (
+      <Alert severity="success">Success: {errorMessage}</Alert>
+    ) : null;
+  };
+
```

Comment:
why did you make this change?

Reply:
I thought we need the sucess component?

Reply:
Yes but not in the `SignUpForm`  though ?